### ISSUE TITLE
url used to fetch full preview of a result is http, but should be https

### ISSUE THREAD

**Issue by jchartrand** [cwrc/dbpedia-entity-lookup#2]

**Comment by jchartrand** [comment]
https://github.com/cwrc/viaf-entity-lookup/issues/3

**Comment by jchartrand** [comment]
@jefferya 
TL;DR Can we change the proxy url from
https://lookup.services.cwrc.ca/dbpediafull
to:
https://dbpeida.lookup.services.cwrc.ca/  
?
Full explanation:
 The call to the new proxy that Jeff created:

https://lookup.services.cwrc.ca/dbpediafull

 works fine, and when set it in an iframe, it returns the html page from dbpedia, but the dbpedia page of course has links to css, jquery libraries, etc. which in this case are links to the root of the domain, and so when the iframe tries to load them, it tries to get them from the domain name from which it thinks it loaded, *without the path*:

https://lookup.services.cwrc.ca

and not from the proxy url we are using, which has a path in addition to the domain name:

https://lookup.services.cwrc.ca/dbpediafull

so a link to load a library file, when the iframe tries to load it, becomes:

https://lookup.services.cwrc.ca/statics/css/bootstrap.min.css

which of course won’t load, because it is missing the dbpediafull part of the path.  If I add the path manually, it does load:

https://lookup.services.cwrc.ca/dbpediafull/statics/css/bootstrap.min.css

So, instead of using a proxy like:

https://lookup.services.cwrc.ca/dbpediafull

if we instead used a proxy like:

https://dbpeida.lookup.services.cwrc.ca/  

with no path, would that work (in your opinion)?  That way when the iframe tries to load the css against the root, it would resolve properly, e.g., to:

https://dbpeida.lookup.services.cwrc.ca/statics/css/bootstrap.min.css

If you agree, could we create the new proxy?  It can be named anything, just without a path.

**Comment by jefferya** [comment]
@jchartrand 

When you have a chance, can you test:
* https://dbpedia.lookup.services.cwrc.ca
* https://getty.lookup.services.cwrc.ca

**Comment by jchartrand** [comment]
Thank you @jefferya <https://github.com/jefferya>

I’ve update the urls in the code, and now just working out a build problem with Andrew, and then hopefully all is well.

**Comment by jchartrand** [comment]
Fixed in https://dev-cwrc-writer.cwrc.ca/
